controllers/providers/metrics/prometheus.go

```diff
@@ -119,42 +140,57 @@ func (m *PrometheusMetrics) Unregister() {
 
 // init instantiates particular metrics
 func (m *PrometheusMetrics) init() {
-	m.metrics.HealthyRecords = prometheus.NewGaugeVec(
+	m.metrics.K8gbGslbHealthyRecords = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Namespace: m.config.K8gbNamespace,
-			Subsystem: gslbSubsystem,
-			Name:      "healthy_records",
-			Help:      "Number of healthy records observed by K8GB.",
+			Name: "k8gb_gslb_healthy_records",
+			Help: "Number of healthy records observed by K8GB.",
 		},
 		[]string{"namespace", "name"},
 	)
 
-	m.metrics.IngressHostsPerStatus = prometheus.NewGaugeVec(
+	m.metrics.K8gbGslbStatusPerIngressHosts = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Namespace: m.config.K8gbNamespace,
-			Subsystem: gslbSubsystem,
-			Name:      "ingress_hosts_per_status",
-			Help:      "Number of managed hosts observed by K8GB.",
+			Name: "k8gb_gslb_status_per_ingress_hosts",
+			Help: "Number of managed hosts observed by K8GB.",
 		},
 		[]string{"namespace", "name", "status"},
 	)
 
-	m.metrics.ZoneUpdateTotal = prometheus.NewCounter(
+	m.metrics.K8gbGslbErrorTotal = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Namespace: m.config.K8gbNamespace,
-			Subsystem: gslbSubsystem,
-			Name:      "delegated_zone_update",
-			Help:      "Number of delegated zone updates. Is hit for create as well",
+			Name: "k8gb_gslb_error_total",
```

Comment:
Consider making counter name plural
```suggestion
			Name: "k8gb_gslb_errors_total",
```